tests/baselines/reference/arrayLiteralAndArrayConstructorEquivalence1.types

```diff
@@ -20,8 +20,8 @@ var myCars3 = new Array({});
 >   : ^^
 
 var myCars4: Array; // error
->myCars4 : any
->        : ^^^
+>myCars4 : Array<any>

```

Comment:
The first iteration of this PR didn't fill missing type arguments (that were otherwise required) with error types. Personally, I like this variant that fills them with error types better: https://github.com/microsoft/TypeScript/pull/59297/commits/e13df6c1c70e55859ff4ac37b0d25cd996097bf1